components/commands/common/key_names.cc

```diff
@@ -0,0 +1,286 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/commands/common/key_names.h"
+
+#include "build/buildflag.h"
```

Comment:
you want `build/build_config.h`.

`build/buildflag.h` only declares `BUILDFLAG` define to be used with actual buildflags, i.e. it doesn't declare `IS_WIN` (what you really want).

Reply:
Cheers, thanks!